content/chainguard/chainguard-images/features/ca-docs/custom-assembly-console/index.md

```diff
@@ -0,0 +1,99 @@
+
+If a build fails, you'll need to make the appropriate changes before attempting another build. You can check their logs for information about what went wrong and what to fix.
+
+
+## Listing Builds and Viewing Logs
+
+You can view a list of all the available builds of your customized container image by clicking the customized image's **Builds** tab in the console:
+
+<center><img src="custom-assembly-3.png" alt="Screenshot of a Custom Assembly container image's Builds tab, with the Builds tab highlighted in a yellow box." style="width:1100px;"></center>
+<br /> 
+
+The table in the Builds tab has six columns:
+
+* **Status**: The status of the given build. When a build is successful, this column will show a green check inside of a circle. When a build has failed, this column will display a red exclamation mark in a triangle.
+* **ID**: A unique identifier representing a specific customized container image build.
+* **Tag**: The container image version the build represents.
+* **Digest**: A unique, content-based hash representing the given container image build. 
+* **Duration**: The amount of time it took to build the container image.
+* **Created**: How long it's been since the build was created. 
+
+Note that if you only recently customized the container image it may take a few minutes for the latest builds to populate.
+
+Additionally, builds will only stay listed in the console for 24 hours. This is because Chainguard Containers, including Custom Assembly container images, are rebuilt frequently and would quickly congest the user interface.
+
+You can click on the row of any build listed in the Builds tab to access its logs. This will cause a window to appear from the right where you can get more details about the build, including build failures:
+
+<center><img src="custom-assembly-4.png" alt="Screenshot of a customized container image build's logs, showing a successful build." style="width:650px;"></center>
+<br /> 
+
+
+## Undoing Changes to a Customized Container Image
+
+If you've made modifications to a customized image and need to revert them, you can do so with just a few clicks in the Chainguard console.
+
+Going back to the container image you just customized, click once again on the **Customize Image**. In the panel where you added packages, there will be a list of the packages added to the customized image below the **Filter packages** search box:
+
+<center><img src="custom-assembly-5.png" alt="Screenshot of the custom-node image's customization menu, before the preview screen. It shows three packages that have been added but can also be removed: bash, curl, and wget." style="width:650px;"></center>
+<br /> 
+
+You can remove these packages from the container image by clicking their respective **⮿** symbols. Removing all the added packages will return the image to its original state.
```

Comment:
Did you miss an image here?

Reply:
I mean "their respective **[]** symbols"

Reply:
ah, I think that's a character set thing. I'll fix that, thanks